### DISCUSSION TITLE
FPGA Select Xilinx's Software?

### DISCUSSION THREAD

**Discussion by Popcorngaemrs** [logisim-evolution/logisim-evolution#1492] [Q&A] [answered]
the fpga menu wants me to select the Xilinx's software but, I don't understand what software it is wants
does anyone know what it is wanting?

**Comment by maehne** [comment, marked as answer]
Depending on the type of FPGA on your development board, you need a different software toolchain to synthesize and map your design on the target FPGA. Logisim-evolution supports currently the following FPGA toolchains:

- Intel Quartus Prime for Altera/Intel FPGAs
- Xilinx Vivado for newer Xilinx/AMD FPGAs (>= Series 7)
- Xilinx ISE for older Xilinx FPGAs/CPLDs (<= Series 6)

When you choose a development board in the FPGA Commander, it will ask you the path to the directory containing the executables needed during synthesis/mapping depending on the target FPGA on the board. Once correctly configured, it remembers that path.

To answer more precisely your question, you need to tell us the development board/target FPGA, which you would like to use.
